### PR TITLE
Give ability to execute scripts

### LINKED ISSUE #13: Error in sqlite3 script execution
While writing the test suit, I discovered that we only execute the first sentence in the `sqlite3` migration system. This is a **high priority fix**.

It seems to come from how the package to interact with sqlite3 is designed.

### PR DESCRIPTION
**Pull request by maiste**
This PR fixes #13  by importing the possibility to execute `SQL script` into the `omigrate-sqlite3` driver.
This is achieved by directly executing the request instead of trying to prepare it.

It requires #12 to be merged first.

### CODE REVIEW

File: example.t/sqlite3/33_create_table.up.sql [hunk 1]
```diff
@@ -1,3 +1,9 @@
 CREATE TABLE pets (
   name string NOT NULL DEFAULT "dog"
 );
+
+CREATE TABLE bidule (
```
Comment (tmattio) [inline example.t/sqlite3/33_create_table.up.sql:5]: Let's find a better name 😉

File: dune [hunk 2]
```diff
@@ -0,0 +1,5 @@
+(cram
+ (applies_to example)
+ (enabled_if
+  (= true %{env:GITHUB_TESTS=false}))
```
Comment (tmattio) [inline dune:4]: I don't understand this: why enable the tests only on the CI?

File: example.t/sqlite3/33_create_table.up.sql (hunk 1 above)
Comment (maiste) [reply example.t/sqlite3/33_create_table.up.sql:5]: Indeed ^^
Would "users" work better?

File: dune (hunk 2 above)
Comment (maiste) [reply dune:4]: We need a container running with Postgres to execute the PostgreSQL test and an SQLite executable available to run the SQLite test. However, `opam-repo-ci` doesn't support these kinds of tests... This is the only option I've found to ensure we don't break `opam-repo-ci` when releasing new versions.

File: example.t/sqlite3/33_create_table.up.sql (hunk 1 above)
Comment (tmattio) [reply example.t/sqlite3/33_create_table.up.sql]: Sure, that sounds good!

File: dune (hunk 2 above)
Comment (tmattio) [reply dune:4]: Thanks! I can't come up with a better alternative either, so let's go with this

**Review by tmattio — APPROVED** [review]

File: dune (hunk 2 above)
Comment (maiste) [reply dune:4]: We can correct it later if we find something better :+1:

**Comment by maiste** [comment]
Thanks for the review and the suggestions! `omigrate.0.3.2` in coming :rocket:

**Pull request merged**